test/performance/odp_ipsecfwd.c

```diff
@@ -1365,8 +1401,6 @@ static odp_bool_t setup_pktios(prog_config_t *config)
 		pktio->num_tx_qs = config->num_output_qs;
 		odp_pktout_queue_param_init(&pktout_param);
 		pktout_param.num_queues = pktio->num_tx_qs;
-		pktout_param.op_mode = config->num_thrs > (int)pktio->num_tx_qs ?
-			ODP_PKTIO_OP_MT : ODP_PKTIO_OP_MT_UNSAFE;
```

Comment:
This looks like a bug fix that probably should be in its own patch before the other patches of this PR.

Reply:
Moved to its own patch (6865d09) in v3.